sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -647,9 +709,12 @@ public virtual Task<Response<string>> CreateRelationshipAsync(string digitalTwin
         /// <param name="relationship">The application/json relationship to be created.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>The http response.</returns>
+        /// <exception cref="RequestFailedException">
+        /// The exception that captures the errors from the service. Check the ErrorCode and Status properties for more details.
+        /// </exception>
         /// <remarks>
         /// <para>Relationships are a one-way link from a source digital twin to another, as described at creation time of the assigned model of the digital twin.</para>
-        /// <para>This argument must be the serialization of an instance of <see cref="Azure.DigitalTwins.Core.Serialization.BasicRelationship"/> or the serialization of an extension of that type.</para>
+        /// <para>This argument must be the serialization of an instance of <see cref="Serialization.BasicRelationship"/> or the serialization of an extension of that type.</para>
```

Comment:
This is inaccurate, must like the one I removed for BasicDigitalTwin. It would be best to just remove it, but if you can think of a better way to say it, that'd be fine.

It doesn't need to be an instnace of BasicRelationship or a derived type. I could hand craft my json, or define my own type.

Reply:
I just updated the namespace as it seemed like Azure.DigitalTwins.Core was not needed to get to the class. I believe this will be removed once I rebase with your changes.

Reply:
I had only removed the line on another method about BasicDigitalTwin. I missed this one, so I'm asking you to remove it.